### ISSUE TITLE
Emby compatability?

### ISSUE THREAD

**Issue by gfish69** [tidusjar/Ombi#2882]
I use Ombi and Emby as Dockers in Unraid. Both with own IP's, but on same subnet. Ombi says Emby connection successful. However, Ombi does not show most of Emby library as available. Should I have stayed with Emby 3?

Emby 4.0.2.0
Ombi 3.0.4256

**Comment by tidusjar** [comment]
Need logs...

There is a issue template when creating an issue on github. Please fill it out.

**Comment by gfish69** [comment]
New to GitHub. I hope this is everything you need. Thanks in advance..

Ombi does not show all Emby TV or Movies as available.

When I log into Ombi as user or admin, and browse to TV or Movie tab, then search for something I know Emby to have, all I get is the request button for most library items. 
Plus a show Ombi does know about I only get the Available button. I thought I remembered a Watch on Emby button at one time.

Screenshots If applicable, add screenshots to help explain your problem.
![image](https://user-images.githubusercontent.com/48441624/54133032-b5eb8f00-43eb-11e9-9ae2-906e137b5da0.png)
![image](https://user-images.githubusercontent.com/48441624/54133075-cdc31300-43eb-11e9-8a49-cb5e1c8a6c9c.png)

Ombi Log File:
[log-20190311.txt](https://github.com/tidusjar/Ombi/files/2952752/log-20190311.txt)
[log-20190305.txt](https://github.com/tidusjar/Ombi/files/2952787/log-20190305.txt)

Sonarr Log:
[sonarr.txt](https://github.com/tidusjar/Ombi/files/2952798/sonarr.txt)

Ombi: 192.168.20.62
Version | 3.0.4256
Branch | master

Emby: 192.168.20.60
 4.0.2.0

Sonarr: 192.168.10.72
ver. 2.0.0.5301

Ombi does successfully load Qualities from Sonarr and Radarr. Ombi using API for each.

**Comment by tidusjar** [comment]
Is Emby enabled in the settings?

**Comment by gfish69** [comment]
Emby is Enabled.

[log-20190311.txt](https://github.com/tidusjar/Ombi/files/2953308/log-20190311.txt)

**Comment by tidusjar** [comment]
Looking at the log it looks like Emby is not configured correctly. 

Please double check your hostname, port and access token

**Comment by gfish69** [comment]
I'm confused. If it Emby settings in Ombi were not correct, would I still get a Successfully  connected response? Wait, does Ombi do its sync locally or over internet. I have verified Hostname/IP, but its a local IP. I twice verified port number 8096. This is Emby standard port. Even checked API from Emby. I will try Emby SSL..

![image](https://user-images.githubusercontent.com/48441624/54152112-d11dc500-4412-11e9-9c73-e3822cb1d9ad.png)

**Comment by tidusjar** [comment]
Yeah it doesn't seem right that you get a success notification, but the logs are telling me it cannot connect. 

We connect to ombi over the details you provide in the settings

**Comment by gfish69** [comment]
So where does this leave me and my issue? I checked Ombi status for Emby latest movie, and it shows available AND I have an view on Emby button! Clicking the View on Emby button, does bring me to Emby login screen, and I am external to my network at the moment. I do connect to Emby and Ombi using NGINX Reverse Proxy Manager, when external. That seems to be working.

**Comment by tidusjar** [comment]
Wait. So if things are now appearing as available, what's the issue?

**Comment by gfish69** [comment]
That is the problem. Most, do NOT appear as available, and only offer to request. Yet some do show available. I also noticed that none of the TV series give you option to play on Emby. Some TV shows show as available, but I get red circle with slash when hovering over available button. Movies are much the same, many I have do not show as on Emby or if they do most only show available. That said I DO have some movies that offer to play in Emby as I believe designed. I have tried Uninstalling Docker and even remove appdata folder. Looking for any ideas...
